python/lsst/ts/FiberSpectrograph/avsFiberSpectrograph.py

```diff
@@ -192,11 +198,18 @@ def _connect(self, serial_number=None):
         self.log.info("Activated connection (handle=%s) with USB device %s.", self.handle, device)
         self.device = device
 
+        # store the number of pixels for use when taking exposures.
+        n_pixels = _getUShortPointer()
+        code = self.libavs.AVS_GetNumPixels(self.handle, n_pixels)
+        assert_avs_code(code, "GetNumPixels")
+        self._n_pixels = n_pixels.contents.value
+
```

Comment:
In the doc string for disconnect please separate the two lines (brief description followed by a detail). Please look at other doc strings as well.

I know this is pre-existing. I apologize for missing it in my last review.